src/main/java/com/feedzai/commons/sql/abstraction/engine/impl/H2Translator.java

```diff
@@ -292,6 +293,29 @@ public String translate(DbColumn c) {
         }
     }
 
+    @Override
+    public String translate(final DbColumnType type) {
+        switch (type) {
+            case BOOLEAN:
+                return "BOOLEAN";
+
```

Comment:
I don't like to put newlines in cases.  
Just a comment, do whatever you want. :)